packages/modules/manage-versions/src/components/contextMenu/ContextMenu.tsx

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SvgMore } from "@itwin/itwinui-icons-react";
+import { DropdownMenu, IconButton, MenuItem } from "@itwin/itwinui-react";
+import React from "react";
+
+export type MenuAction = {
+  icon?: JSX.Element;
+  onClick: () => void;
+  title: string;
+  label: string;
+  disabled?: boolean;
+};
+
+export interface ContextMenuProps {
+  menuActions: MenuAction[];
+  isMenuOpen: boolean;
+  toggleMenu: (rowId: string) => void;
+  rowId: string;
```

Comment:
whats the point to pass here `rowId` if you just pass it back?

Reply:
@veekeys Removed controlled logic for Dropdown component. Hence, no need of this prop at all.